.vscode/settings.json

```diff
@@ -82,9 +82,12 @@
         "variant": "cpp"
     },
     "python.autoComplete.extraPaths": [
+        "/home/arthur/WRover_Software/devel/lib/python3/dist-packages",
```

Comment:
Relative path?
```suggestion
        "${workspaceFolder}/devel/lib/python3/dist-packages",
```

Reply:
Addressed in [277b00a](https://github.com/WisconsinRobotics/WRover_Software/pull/48/commits/277b00a47c8537aba6ba2aeeb6c09d9b5ea4d3eb).